src/stores/FilterBarStore.js

```diff
@@ -150,11 +156,7 @@ export class FilterBarStore {
   }
 
   disableAllFilters() {
-    var enabledFilters = this.getEnabled();
-
-    for (var filterUid in enabledFilters) {
-      this.disableFilter(filterUid);
-    }
+    this.activeFilters = []
```

Comment:
So we don't need to call disable filter on each individually now?

Reply:
Yes, because architecture and data-structure changed.